core/vtk/ttkIdentifiers/ttkIdentifiers.h

```diff
@@ -1,31 +1,24 @@
 /// \ingroup vtk
 /// \class ttkIdentifiers
-/// \author Julien Tierny <julien.tierny@lip6.fr>
-/// \date August 2015.
+/// \author Eve Le Guillou <eve.le-guillou@lip6.fr>
```

Comment:
in principle (mostly for future reference), do not erase the name of previous contributors but just add yours to the list.